README.md

```diff
@@ -21,3 +21,19 @@ The GitLab runner is an EC2 instance that can use the KMS key connected to its e
 to read secrets from the parameter store and interact with the Aurora cluster.
 
 [architecture]: infra-arch.svg "iAtlas architecture"
+
+## API Deployment
+
+The API is a Docker container built in Gitlab CI, so the deployment via Elastic Beanstalk requires two files:
+
+- The `Dockerrun.aws.json` file that specifies the application parameters to Docker
+- The `iatlas-[staging|production]-dockercfg` file that tells Docker how to authenticate against the Gitlab container registry
+
+These need to live in an S3 bucket, typically with a key prefix of the environment name (e.g. `staging/Dockerrun.aws.json`) and have ACLs on them that allow Elastic Beanstalk to read them. Examples of these files are available in the [iAtlas-API](https://gitlab.com/cri-iatlas/iatlas-api) repo.
+
+This also assumes that a secret exists in SecretsManager that contains the RDS creds for a non-root user for the application to use. CloudFormation can't manipulate user accounts in RDS, so this needs to be done manually:
```

Comment:
secrets and secrets manager are a big bundle of goop to deal with.  There's just not enough in this PR to even setup secrets.  First there's the KMS key, second there's the policies, and third there's how you want to allow your  instances to access the secrets.